home_work/hw_14/calculator/calculator.py

```diff
@@ -0,0 +1,55 @@
+from loguru import logger
+
+
+def evaluate_expression(expression):
+    try:
+        expression = expression.replace("\\", "/")
+
+        return eval(expression)
```

Comment:
if you are going to use `evel()` add validator

Reply:
add simple validator . done